cmd/vic-machine/create/create.go

```diff
@@ -1292,61 +1312,80 @@ func logArguments(cliContext *cli.Context) {
 				continue
 			}
 			url.User = nil
-			log.Debugf("--target=%s", url.String())
+			flag := fmt.Sprintf("--target=%s", url.String())
+			log.Debug(flag)
+			args = append(args, flag)
```

Comment:
no, it's for all options. For example, https://github.com/vmware/vic/blob/master/cmd/vic-machine/create/create.go#L179, cli will set a default value for this option, if we do not compare the default value, I'm afraid it will persist almost all options